Scripts/Python/ki/xKIChat.py

```diff
@@ -952,11 +952,45 @@ def __call__(self, message):
                         except:
                             PtDebugPrint("xKIChat.commandsProcessor(): Chat command function did not run.", command, level=kErrorLevel)
                 except LookupError:
-                    if str(words[0].casefold()) in xKIExtChatCommands.xChatSpecialHandledCommands:
+                    firstWordLower = str(words[0].casefold())
```

Comment:
```suggestion
                    firstWordLower = words[0].casefold()
```

Yeah, it was goofy before, but let's go ahead and fix it.